src/ripple/protocol/TxMeta.h

```diff
@@ -101,6 +101,11 @@ class TxMeta
     {
         return (mNodes);
     }
+    STArray const&
+    peekNodes() const
```

Comment:
It wouldn't hurt to just call this `getNodes` too.

Reply:
oh yeah, for some reason I thought this wouldn't compile. Let me change that.